server/lib/paas.coffee

```diff
@@ -46,7 +46,8 @@ class AppManager
 
     # Get token from token file if in production mode.
     getAuthController: ->
-        if process.env.NODE_ENV is 'production'
+        if process.env.NODE_ENV is 'production' or
```

Comment:
Yes, it's a bit easier to read. But I prefered to be coherent with the rest of cozy-controller. For example, we have the same check here: https://github.com/cozy/cozy-home/blob/master/server/models/application.coffee#L36